packages/legend-art/style/base/_modal.scss

```diff
@@ -55,7 +55,7 @@
     @include flexVCenter;
   }
 
-  &__header__action {
+  &__header__action-btn {
```

Comment:
let's revert this change, action sometimes doesn't have to be button.

Reply:
! true